crates/re_sdk/src/recording_stream.rs

```diff
@@ -1528,10 +1541,57 @@ impl RecordingStream {
         ThreadInfo::set_thread_time(
             &this.info.store_id,
             Timeline::new(timeline, TimeType::Time),
-            ns.into().map(|ns| Time::from_ns_since_epoch(ns).into()),
+            Time::from_ns_since_epoch(ns.into()).into(),
         );
     }
 
+    /// Clears out the current time of the recording for the specified timeline, for the
+    /// current calling thread.
+    ///
+    /// This will clear out _both sequential and temporal_ timelines of the specified name.
+    /// Refer to [`Self::disable_timeline_specific`] if you need more fine-grained control.
```

Comment:
Oof, I didn't even realize we could have a sequential and temporal timeline with the same name. I think this is a bug rather than a feature. The timeline _name_ should be a unique identifier for the timelines.

In other words: `TimePoint` should really map `TimelineName` to a (`TimeType, TimeInt`) pair. I don't think we need to make that change right away, but we should present an API that suggests that timeline names may not clash. In this case, the easy way it so make `disable_timeline_specific` private and remove the docs linking to it, or just make `disable_timeline` disable all timelines with that name.